docs/contributing/testing/Writing-a-golden-file-test-for-package-flutter.md

```diff
@@ -50,6 +51,88 @@ Notice, Gold may wrongly attribute blame for image changes on the dashboard. Pos
 
 ## Creating a New Golden File Test
 
+<!--
+  Note: These instructions are also referred to in the flutter/engine repository
+  ensure that when updating them, the workflow is either consistent or the
+  differences are clearly called out.
+-->
+
+Writing a _new_ golden file test is an involved process, particularly if the
+_test suite_ (sometimes called "task" or "shard") is not already configured for
+golden file testing, and missing a step can result in a broken build or a
+failing test.
+
+The process involves:
+
+1. (Potentially) [Adding or configuring a `.ci.yaml` task](#add-or-configure-a-ciyaml-task).
+1. [Writing and submitting the test](#writing-and-submitting-the-test).
+1. (Potentially) [Adding a known task name to the `flutter-gold` check](#adding-a-known-task-name-to-the-flutter-gold-check).
```

Comment:
> (Potentially)

All parenthetical `potentially`s should be removed. They read as incomplete docs that don't help.